### ISSUE TITLE
Constraint.ArtPen.ArtPen_WellName    Trigger: Update only

### ISSUE THREAD

**Issue by CanDKD** [agrc/uic-attribute-rules#99]
I played around with different settings for this rule.  It only raises an error when the trigger is set to Update. Trigger set to Insert only won't raise the error at all.   Insert and Update will not allow the record to be created.

I am going to create some ARs for the other manual entry fields that can't be empty based on this rule.  ContactName, ContactOrganization, ContactMailAddress, etc.  I will copy the arcade expression here after testing them.

**Comment by steveoh** [comment]
Here is well name... https://github.com/agrc/uic-attribute-rules/blob/master/src/rules/arcade/artPen/wellNameConstraint.js

Check the other rule folders before duplicating any work please
https://github.com/agrc/uic-attribute-rules/tree/master/src/rules/arcade

**Comment by CanDKD** [comment]
I have only been Disabling or changing triggers to see what works.  I am not messing with your Arcade scripts.  When  I want to try my hand at creating an Arcade expression, I create a new rule and copy some of your code to see if I can make it work.  I am only proposing to create rules that are tagged as 'enhancements' as I feel these were above and beyond what we agreed to for costing out this project.

**Comment by CanDKD** [comment]
I have changed my mind.  I am not going to try to create ARs from my end.  I tried creating a simple Constraint.Contact.Name based on your Constraint.ArtPen.ArtPen_WellName both set to trigger on update.  Your ArtPen_WellName AR worked fine after I set the trigger to update only and all the other ARs continued to work too.  When I created a new AR Constraint.Contact.Name it worked but the other two that you created stopped working.  I'll leave all the AR work to you!!
